dev/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM quay.io/astronomer/astro-runtime:5.0.6-base
+
+USER root
+RUN apt-get update -y && apt-get install -y git
+RUN apt-get install -y --no-install-recommends \
+        build-essential \
+        libsasl2-2 \
+        libsasl2-dev \
+        libsasl2-modules
+ENV SETUPTOOLS_USE_DISTUTILS=stdlib
+
+COPY ../ .
+COPY . ${AIRFLOW_HOME}/astro_sdk/
```

Can this not be clubbed into one?
```suggestion
COPY ../  ${AIRFLOW_HOME}/astro_sdk/
```